hazelcast-client/src/test/java/com/hazelcast/client/discovery/AwsCloudDiscoveryTest.java

```diff
@@ -119,16 +123,25 @@ public void testAwsAddressProvider() {
                 .setTagKey(AWS_TEST_TAG)
                 .setTagValue(AWS_TEST_TAG_VALUE);
 
+
         ClientConfig clientConfig = new ClientConfig();
-        clientConfig.getNetworkConfig().setAwsConfig(clientAwsConfig);
+        String instanceName = null;
+        if (clientConfig.getInstanceName() != null) {
+            instanceName = clientConfig.getInstanceName();
+        } else {
+            instanceName = "hz.client_testAwsAddressProvider";
+        }
 
-        HazelcastInstance client = HazelcastClient.newHazelcastClient(clientConfig);
-        IMap<Object, Object> map = client.getMap("MyMap");
-        map.put(1, 5);
-        assertEquals(5, map.get(1));
+        GroupConfig groupConfig = clientConfig.getGroupConfig();
+        String loggingType = clientConfig.getProperty(GroupProperty.LOGGING_TYPE.getName());
+        LoggingService loggingService = new ClientLoggingService(groupConfig.getName(),
+                loggingType, BuildInfoProvider.getBuildInfo(), instanceName);
 
-        AwsAddressProvider awsAddressProvider = new AwsAddressProvider(clientAwsConfig, client.getLoggingService());
+        AwsAddressProvider awsAddressProvider = new AwsAddressProvider(clientAwsConfig, loggingService);
         Collection<Address> addresses = awsAddressProvider.loadAddresses();
         assertTrue(isNotEmpty(addresses));
```

Comment:
A failure on `assertTrue()` gives a quite meaningless error. We could improve this, maybe like this
```java
assertTrue("Expected non-empty addresses from AwsAddressProvider.loadAddresses()", isNotEmpty(addresses));
```

Reply:
done.